content/news/2025-01-03-abstract-review/index.en.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Volunteer as an R-Ladies Abstract Reviewer!"
+author: "Abstract Review Team"
+type: blog
+date: "2025-01-03"
+image: img/global_team.png
+tags: 
+- community
+categories: 
+- R-Ladies
+summary: Volunteer as an R-Ladies Abstract Reviewer to support diversity in tech!
+slug: abstract-review-2025
+---
+
+Are you passionate about supporting diversity in tech, data science, and academia? Want to help aspiring speakers craft compelling abstracts for conferences? Volunteer to be a reviewer for the R-Ladies Abstract Review System and make a difference in creating a more inclusive and representative speaker lineup at key industry and academic events!
+
+## What is the R-Ladies Abstract Review System?
+
+Our abstract review system connects aspiring speakers with experienced reviewers who can provide feedback and support. By volunteering as a reviewer, you'll play a critical role in helping individuals from underrepresented groups take their first steps into public speaking or advance their speaking careers.
+
+## Why Join as a Reviewer?
+
+* Make an Impact: Help new voices shine by sharing your expertise and guidance.
+* Flexible Commitment: Sign up now, and we'll reach out when abstracts come in.
+* Support a Global Movement: Contribute to R-Ladies' mission of promoting diverse voices in the R community.
+* Minimal Time Commitment: Totally flexible, depending on the level of feedback you wish to provide and number of abstracts you wish to review.
+* Expand Your Network: Meet new people from around the world and learn about their work.
+
+
+## Who Can Be a Reviewer?
+
+Anyone with experience writing or reviewing conference abstracts, giving talks, or attending conferences is encouraged to sign up. Our reviewers come from diverse backgrounds and skill levels, and we welcome all genders to participate.
+
+## How to Sign Up
+
+[Sign up here](https://airtable.com/appJadVolZxoDGSIK/pag4bpfeGIATQFefk/form) to join the R-Ladies Abstract Review Team. Even if you're not sure about your availability, signing up ensures you'll receive updates and opportunities to contribute when abstracts are ready for review.
+
+## What Happens Next?
+
+Once you sign up:
+
+1. We'll reach out as abstract review opportunities arise which you can accept or decline depending on your availability.
+2. You'll be part of a supportive network making a tangible difference in the R and tech communities.
+
+Join us today and help shape the future of conference lineups—one abstract at a time!
+
```

Comment:
What about commitment length? Are they committing to be available for a year? two? will they have an opportunity to opt out? i'm probably being too picky though and it is probably fine as is, feel free to ignore.

Reply:
I think it's important - updated to say 1 year and that they can opt out at any time.  Then we can repeat this in a year and ensure we don't waste time assigning abstracts to reviewers who no longer want to participate.